### DISCUSSION TITLE
I tried this.

### DISCUSSION THREAD

**Discussion by mstrauss2018** [nextcloud/all-in-one#3988] [Questions] [answered]
I tried this.
when I ran borg mount it asked for my passphrase 3 times. Each time I pasted it in.
I wen to the AIO interface and copied the encryption password for backups listed under the Backup & Restore section.
every file I ran the cat command on only listed the word Putty many times.
Did it ask for the passphrase 3 times because it was the wrong one? If so, then where do I get my passphrase?
I've been trying for over 1 week to try to restore 1 file and this is the closest I've gotten so far.

_Originally posted by @mstrauss2018 in https://github.com/nextcloud/all-in-one/discussions/3330#discussioncomment-7945719_

**Comment by szaimen** [comment, marked as answer]
Is getting answered in https://help.nextcloud.com/t/need-help-recovering-an-individual-file-from-borg-backup/177696
